install.sh

```diff
@@ -1,81 +1,166 @@
 #!/bin/bash
-GOODCOLOR='\033[1;36m'
-WARNCOLOR='\033[1;31m'
+
 NOCOLOR='\033[0m'
-GOODPREFIX="${GOODCOLOR}INSTALLER:"
-WARNPREFIX="${WARNCOLOR}ERROR:"
+LOGCOLOR='\033[1;36m'
+PASSCOLOR='\033[1;32m'
+WARNCOLOR='\033[1;31m'
 
-# Sane installation defaults for no argument cases
-CATKIN_DIR=~/sub_ws
-REQUIRED_OS="trusty"
+LOGPREFIX="${LOGCOLOR}INSTALLER:"
+WARNPREFIX="${WARNCOLOR}ERROR:"
+PASSTEXT="${PASSCOLOR}PASS"
+FAILTEXT="${WARNCOLOR}FAIL"
 
 instlog() {
-    printf "$GOODPREFIX $@ $NOCOLOR\n"
+    printf "$LOGPREFIX $@ $NOCOLOR\n"
```

Comment:
Although convenience is also good, which is probably better than following standards